scripts/scrape.py

```diff
@@ -10,8 +10,9 @@
 
 import scrapers  # noqa: E402
 
-log = logging.getLogger(__name__)
-logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig()
+log = logging.getLogger('purview')
+log.setLevel(logging.DEBUG)
```

Comment:
you can actually include logging level in the logging.basicConfig() statement

Reply:
If I do that though, then all other loggers will inherit that log level.